### PR TITLE
Clean up InternalFrame and around anchor.

### PR DESCRIPTION
**Pull request by ueshin**
Clean up `InternalFrame` to remove `spark_column` and fix around it, also add some utility methods.

### CODE REVIEW

File: databricks/koalas/series.py [hunk 1]
```diff
@@ -2829,7 +2818,7 @@ def transpose(self, *args, **kwargs):
         2    3
         Name: 0, dtype: int64
         """
-        return first_series(DataFrame(self._internal.copy(spark_column=None)))
+        return self.copy()
```
Comment (itholic) [inline databricks/koalas/series.py:2821]: +1

**Comment by itholic** [comment]
LGTM

**Comment by HyukjinKwon** [comment]
@ueshin Feel free to merge. I will not merge in case you have some more things to push here.

**Comment by ueshin** [comment]
I'd merge this now. I'll have some more things to do but they should be in a separate PR.

**Pull request merged**